tests/dummy/app/pods/client-typed/template.hbs

```diff
@@ -41,7 +41,9 @@
   controls=(component 'frost-action-bar'
     hook='clientActionBar'
     selectedItems=selectedItems
+    isLoading=isLoading
     itemTypeKey='itemType'
+    loadingText=loadingText
```

Comment:
Please add this property to the README.

Reply:
I have added a table to the README which describes this property.